packages/connector-alipay/jest.config.ts

```diff
@@ -0,0 +1,20 @@
+import type { Config } from '@jest/types';
+
+const config: Config.InitialOptions = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  moduleNameMapper: {
+    '^@/(.*)$': '<rootDir>/src/$1',
+    '^jose/(.*)$': '<rootDir>/node_modules/jose/dist/node/cjs/$1',
```

Comment:
no reference?

Reply:
It looks like this line is copied from other place, need to remove.